build.yaml

```diff
@@ -39,7 +39,7 @@ build-env:
     tar -xf /stacker/v2_03_15.tar.gz
     cd lvm2-*
     ./configure --enable-static_link
-    make install_device-mapper
+    make -j$(grep -c processor /proc/cpuinfo) install_device-mapper
```

Comment:
fwiw, this just silently-ish swallows failure of the 'grep' command.

better would be:

```sh
numcpu=$(grep -c processor /proc/cpuinfo)
make -j${numcpu} install_device-mapper
```

example:

```
$ cat /tmp/foo
#!/bin/sh
set -e
echo make -j$(bogus command) foo
echo still here

$ sh -x /tmp/foo
+ set -e
+ bogus command
/tmp/foo: 3: bogus: not found
+ echo make -j foo
make -j foo
+ echo still here
still here
```

you end up calling make like `make -j install_device-mapper`.  make will probably fail, but its just odd.

Reply:
fixed